### PR TITLE
Feature Added : Idle Animation

### PR DESCRIPTION
**Pull request by abhi-dave**
Idle animation Feature added in the macOS app.
- Added Menu Item to toggle ideal animation.
- Added Timer which animates every 30 seconds after the Clippy is idle.

### CODE REVIEW

File: Clippy macOS/AgentViewController.swift [hunk 1]
```diff
@@ -11,8 +11,7 @@ import AppKit
 class AgentViewController: NSViewController {
     var agentController: AgentController
     var agentView: AgentView
-    var isZoomed: Bool = true
-    
+    var idleTimer:Timer!
```
Comment (Cosmo) [inline Clippy macOS/AgentViewController.swift:15]: `var isZoomed: Bool = true` is missing and breaks the code. Can you please add it back? :)
And I would appreciate it, if you could add a space after the `:`.
Thanks!

**Review by Cosmo — CHANGES_REQUESTED** [review]
Haha, thats so cool! :D
Would you mind, making these two small fixes? 
Other than that, it looks ready to ship! 🎉

File: Clippy macOS/AgentViewController.swift (hunk 1 above)
Comment (abhi-dave) [reply Clippy macOS/AgentViewController.swift:15]: Darn it! it must've been deleted when I was resolving conflict! 🤦‍♂  missing var is now added please check

**Comment by Cosmo** [comment]
Great work — merged!

Celebration

**Pull request merged**